### PR TITLE
Consolidate examples

### PR DESCRIPTION
**Pull request by johnktims**
This PR moves querydsl-customer-dao, querydsl-sql-example, and querydsl-jpa-example under querydsl/querydsl/querydsl-examples.

### CODE REVIEW

**Comment by johnktims** [comment]
I need to upgrade querydsl-sql-example and querydsl-jpa-example to use querydsl 4.0.1 and add the examples module to the travis profile.

**Comment by timowest** [comment]
This looks good to me.

File: querydsl-examples/querydsl-example-jpa-guice/pom.xml [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+  <packaging>jar</packaging>
+  <name>Querydsl example - JPA Guice</name>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <failIfNoTests>false</failIfNoTests>
+
+    <hibernate.version>4.1.7.Final</hibernate.version>
+    <hibernate.validator.version>4.3.0.Final</hibernate.validator.version>
+    <querydsl.version>4.0.1</querydsl.version>
```
Comment (Shredder121) [inline querydsl-examples/querydsl-example-jpa-guice/pom.xml]: I would just remove all mentionings of `querydsl.version`.

**Comment by Shredder121** [comment]
So does this mean that the original repositories can be removed?
Should we add a new label for `example` or something?

**Comment by johnktims** [comment]
I think we should remove the original repositories.

I'm not sure a new label is necessary but I guess it wouldn't hurt to create one.

**Comment by timowest** [comment]
I agree, we can remove the original repositories.

**Comment by Shredder121** [comment]
Then I'll merge this.

**Pull request merged**